docs/circuit-examples/full-design-flow-examples/hfss_eig_f_convergence.csv

```diff
@@ -0,0 +1,13 @@
+"Pass []","re(Mode(1)) [g]","re(Mode(2)) [g]"
```

Comment:
not sure this file needs to exist, does it? I believe this file is just an intermediate by-product, so it can be just ignored (like the correspondent one in tutorials already is in the .gitignore (`tutorials/**/*.csv`)

Reply:
Good point.  I made a tweak to ignore those files of files and removed it.  Of course, this blew away your approval.